grandalf/routing.py

```diff
@@ -28,8 +28,16 @@ def setpath(self, pts):
 #  tail and head.
 def route_with_lines(e, pts):
     assert hasattr(e, "view")
-    tail_pos = intersectR(e.v[0].view, topt=pts[1])
-    head_pos = intersectR(e.v[1].view, topt=pts[-2])
+    try:
+        tail_pos = intersectR(e.v[0].view, topt=pts[1])
+    except ValueError:
+        # TODO: Create a test case to reproduce it and submit upstream.
```

Comment:
As a note, I'm having this error on some models... if you want I can remove it from the pull request.